### PR TITLE
add Simplified Chinese(2% translated, to be continue. Need Help.)

### PR DESCRIPTION
**Pull request by kisscelia**
1. add i18n --- Simplified Chinese
2. add SASS_BINARY files in the app directory
3. add Dockerfile-cn for people in China.

### CODE REVIEW

**Comment by AmsterGet** [comment]
Hello @kisscelia !
Thank you for your contribution!
Our next version is currently in development and kept in the `develop` branch, so I changed the target, please resolve conflicts accordingly.
We can add your translations to the next version 5.8 and I think we can publish it not fully translated so that you and the community can do this work gradually step by step.
Could you also make a `cn.json` file based on the original English texts? This will help people who do not know Russian better understand what a particular sentence means.
I will review your changes in more detail soon to speed up the merge of this PR.

**Comment by kisscelia** [comment]
> Hello @kisscelia ! Thank you for your contribution! Our next version is currently in development and kept in the `develop` branch, so I changed the target, please rebase under it and resolve conflicts accordingly. We can add your translations to the next version 5.8 and I think we can publish it not fully translated so that you and the community can do this work gradually step by step. Could you also make a `cn.json` file based on the original English texts? This will help people who do not know Russian better understand what a particular sentence means. To do this, you can remove the strings translated into Russian and run the `manage:translations` script from `package.json` - it will fill the file with English content by default. I will review your changes in more detail soon to speed up the merge of this PR.

Hi, @AmsterGet 

Thanks for your reply.

I want you to send me the files (cn.json which generated by manage:translations)  (can send as an attachment via email: sinecelia.wang@gmailcom), because i got an error. I am very very sorry.

image


So, I can begin to translate ASAP.

:-(

**Comment by kisscelia** [comment]
got an error too:
image

**Comment by kisscelia** [comment]
> Hello @kisscelia ! Thank you for your contribution! Our next version is currently in development and kept in the `develop` branch, so I changed the target, please rebase under it and resolve conflicts accordingly. We can add your translations to the next version 5.8 and I think we can publish it not fully translated so that you and the community can do this work gradually step by step. Could you also make a `cn.json` file based on the original English texts? This will help people who do not know Russian better understand what a particular sentence means. To do this, you can remove the strings translated into Russian and run the `manage:translations` script from `package.json` - it will fill the file with English content by default. I will review your changes in more detail soon to speed up the merge of this PR.

Hi, @AmsterGet 

Thanks for your reply.

I am working on it. Thank you.

It may take a lot of time. 

:-)

**Comment by kisscelia** [comment]
1. untranslated items in zh.json are set to english by default.
2. translating, please wait......

File: Dockerfile-cn [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    && npm run build \
+    && npm run test
+
+FROM nginx:alpine
+COPY --from=build-frontend /usr/src/app/build /usr/share/nginx/html
+COPY --from=generate-build-info /usr/src/app/build /usr/share/nginx/html
+RUN rm /etc/nginx/conf.d/default.conf
+COPY nginx.conf /etc/nginx/nginx.conf
+
+EXPOSE 8080
```
Comment (AmsterGet) [inline Dockerfile-cn:35]: Could you explain why our `Dockerfile-full` is not suitable for you?
We use this file for development purposes only. `Dockerfile` is used to run the latest version of the application, which is sufficient in most cases.

**Review by AmsterGet — COMMENTED** [review]
Also please resolve conflicts with the target branch.

File: Dockerfile-cn (hunk 1 above)
Comment (kisscelia) [reply Dockerfile-cn:35]: I got a lot of errors when i use Dockerfile-full, so i changed something(Dockerfile-cn)
I will delete it, and use it locally.

**Comment by kisscelia** [comment]
> 1. untranslated items in zh.json were set to english by default.
> 2. translating, please wait......

I've a lot of work to do, and nearly no time to translate, sorry. :-(

File: Dockerfile-cn (hunk 1 above)
Comment (kisscelia) [reply Dockerfile-cn]: Hi, @AmsterGet 

deleted the files, thanks.

and another question: do i need to add zh. to the file(app/.storybook/config.js)?

Comment (AmsterGet) [reply Dockerfile-cn]: Hello @kisscelia !
Could you please also make a rebase under develop branch to remove unnecessary commits from the PR?

**Comment by AmsterGet** [comment]
I suppose we can publish it partially translated after review to allow other contributors continue work on this.

File: Dockerfile-cn (hunk 1 above)
Comment (kisscelia) [reply Dockerfile-cn]: hard to me to do this,  :-(

I want to re-fork the repo and make a PR under the latest code of the develop branch...

**Pull request closed without merging**